static/templates/items/item-spell-sheet.hbs

```diff
@@ -279,13 +279,19 @@
                 <a class="item-checkbox" data-target="system.overcast.valuePerOvercast.bonus">{{localize 'Bonus'}}<i
                     class="far fa-circle"></i></a>
               {{/if}}
+            {{/if}}
+            <div class="form-group">
+              <label class="label-text">Koszt zmiany</label>
```

Comment:
Needs translation (if it is to remain, look previous comment)

Reply:
done